.github/integration/tests/tests.sh

```diff
@@ -173,6 +171,78 @@ else
     exit 1
 fi
 
+# Test multiple pub key encryption
+
+# Create another couple of key-pairs
+for c in 1 2
+do
+    if ( echo "" | ./sda-cli createKey sda_key$c ) ; then
+        echo "Created key pair for encryption"
+    else
+        echo "Failed to create key pair for encryption"
+        exit 1
+fi
+done
+
+# Create file with concatenated pub keys
+cat sda_key1.pub.pem sda_key2.pub.pem > sda_keys
+
```

Comment:
Maybe here move the creation of the concatenated pub keys to the section where you actually encrypt with this concatenated key (line 211)?